### DISCUSSION TITLE
Api do protheus

### DISCUSSION THREAD

**Discussion by TellesDev** [community/community#50598] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Api do protheus  mata103

**Comment by CaioSilveiraTelles** [comment, marked as answer]
https://centraldeatendimento.totvs.com/hc/pt-br/articles/360006469332-Cross-Segmentos-Totvs-Backoffice-Protheus-SIGACOM-Pontos-de-entrada-do-Documento-de-Entrada-MATA103
